.github/ISSUE_TEMPLATE/release.md

```diff
@@ -57,6 +57,8 @@ I have:
   [releases.crossplane.io] `stable` channel ([build][stable-build]).
 - [ ] Published a [new release] for the tagged version, with the same name as
   the version and descriptive release notes ([release][release]).
+- [ ] Updated the [releases table] in the `README.md` on `master`
+  ([PR][release-table-pr]).
```

Comment:
this looks like it's supposed to be a link, but i don't see it as a link in the preview of this file.  Is this link working as you expect?

Reply:
@jbw976 it is -- this is true for a number of the links in this template as it is expected that the release manager will populate them (per commented instructions at top and bottom of this section), with the appropriate link for the release 👍🏻

Reply:
ah, i missed that the link target was intentionally a commented out example link :)